src/components/TaskDetailsModal/TaskDetailsModal.js

```diff
@@ -40,61 +39,70 @@ class TaskDetailsModal extends PureComponent<Props> {
     isRunning ? abortTask(task, timestamp) : runTask(task, timestamp);
   };
 
-  getStatusText = () => {
-    const { status, timeSinceStatusChange } = this.props.task;
+  renderPrimaryStatusText = () => {
+    const { status } = this.props.task;
 
     switch (status) {
-      case 'idle': {
+      case 'idle':
         return (
-          <span>
+          <PrimaryStatusText>
             Task is <strong>idle</strong>.
-            {timeSinceStatusChange && (
-              <LastRunText>
-                Last run:{' '}
-                {moment(timeSinceStatusChange).format(
-                  'MMMM Do, YYYY [at] h:mm a'
-                )}
-              </LastRunText>
-            )}
-          </span>
+          </PrimaryStatusText>
         );
-      }
-      case 'pending': {
+
+      case 'pending':
         return (
-          <span>
+          <PrimaryStatusText>
             Task is{' '}
             <strong style={{ color: COLORS.orange[500] }}>running</strong>...
-          </span>
+          </PrimaryStatusText>
         );
-      }
 
-      case 'success': {
+      case 'success':
         return (
-          <span>
+          <PrimaryStatusText>
             Task{' '}
             <strong style={{ color: COLORS.green[700] }}>
               completed successfully
             </strong>.
-            <LastRunText>
-              {moment(timeSinceStatusChange).calendar()}
-            </LastRunText>
-          </span>
+          </PrimaryStatusText>
         );
-      }
 
-      case 'failed': {
+      case 'failed':
         return (
-          <span>
+          <PrimaryStatusText>
             Task <strong>failed</strong>.
-            <LastRunText>
-              {moment(timeSinceStatusChange).calendar()}
-            </LastRunText>
-          </span>
+          </PrimaryStatusText>
+        );
+
+      default:
+        throw new Error('Unrecognized status in TaskDetailsModal.');
+    }
+  };
+
+  renderTimestamp = () => {
+    const { status, timeSinceStatusChange } = this.props.task;
```

Comment:
`timeSinceStatusChange` is null for idle tasks causing an invalid date.

<img width="628" alt="screen shot 2018-08-21 at 11 53 20 am" src="https://user-images.githubusercontent.com/17421347/44422736-d4b23300-a538-11e8-89e7-d0ca2475adfd.png">

Reply:
Whoops, forgot about this. Tackling now.